### ISSUE TITLE
GoPhish - remove groups not removing underlying targets

### ISSUE THREAD

**Issue by sietsef** [gophish/gophish#1398]
When i remove a group while using the mysql database the group gets removed but the underlying targets will still exist within the database. Im trying this while using the gophish UI and API

**Comment by S0larflare** [comment]
Yes, I believe this is fully intended as a user could be a member of several groups. For example you could have groups for managers, IT and all company, of which the IT manager would be part of all 3.

**Comment by sietsef** [comment]
> Yes, I believe this is fully intended as a user could be a member of several groups. For example you could have groups for managers, IT and all company, of which the IT manager would be part of all 3.

True, but when i remove all groups why does the user still exist? if you could reuse it, it would make sense but its gone within the UI but still stays within the database

**Comment by S0larflare** [comment]
Ah I see what you mean. I would suspect that is so that it doesn't invalidate any old results, but Jordan would have a better idea, it might just be because SQLite is a bit of a pain in the arse :smile:

**Comment by sietsef** [comment]
Yeah i was also thinking maybe it keeps the targets because of the campaign results, but ive also deleted all of those and the targets are still there. I am using a MySql database
